schemas/ihcl/dialogHeader.ts

```diff
@@ -0,0 +1,47 @@
+const dialogHeaderVariants = [
+  {
+    title: "Default Dialog Header",
+    value: "default-dialog-header",
+  },
+  {
+    title: "Dialog Header With Image Assets",
+    value: "dialog-header-with-image-assets",
+  },
+];
+
+export default {
+  name: "dialogHeader",
+  title: "Dialog Header",
+  type: "object",
+  fields: [
+    {
+      title: "Title",
+      name: "title",
+      type: "string",
+    },
+    {
+      name: "variant",
+      title: "Variant",
+      type: "string",
+      options: {
+        list: [...dialogHeaderVariants],
+      },
+    },
+    {
+        name: "logo",
+        title: "Logo",
+        type: "image",
+      },
+    {
+      title: "Image",
+      name: "imageAsset",
+      type: "imageAsset",
+    },
+    {
+      title: "Language List",
+      name: "languageList",
```

Comment:
What is this? Please provide the feasibility of adding a placeholder here. Make it generic.